src/components/widgets/android-download-button/AndroidDownloadButton.tsx

```diff
@@ -0,0 +1,14 @@
+'use client';
+
+export function AndroidDownloadButton() {
+  return (
+    <a
+      href={process.env.NEXT_PUBLIC_APP_LINK as string}
+      target="_blank"
+      rel="noopener noreferrer"
+      className="whitespace-nowrap rounded-3xl bg-[#9985CB] px-4 py-2 font-bold text-background text-black hover:bg-[#9985CB]/80 hover:duration-200 dark:bg-[#48347A] dark:text-white"
+    >
+      Android
+    </a>
+  );
+}
```

Comment:
this component is the same as IosDownloadButton. I'd suggest making it reusable by passing props.

Additionally, the link for both platforms is `https://link-to.app/sismoMx`
 
@DerekRdgz  @andonirdgz